jsscripts/OrientationChangeHandler.jsm

```diff
@@ -18,9 +18,7 @@ XPCOMUtils.defineLazyModuleGetter(this, "Logger",
                                   "chrome://embedlite/content/Logger.js");
 
 this.OrientationChangeHandler = function OrientationChangeHandler(window) {
-  this.docShell = window.QueryInterface(Ci.nsIInterfaceRequestor)
-                    .getInterface(Ci.nsIWebNavigation)
-                    .QueryInterface(Ci.nsIDocShell);
+  this.docShell = window.docShell;
```

Comment:
This should be fine. Could you please file a task about fixing other nsIDocShell interface requests.

Reply:
Fixed here https://github.com/sailfishos/embedlite-components/pull/16